go/prometheusexporter/collector/collector.go

```diff
@@ -310,98 +422,125 @@ func (c *Collector) registerAjaxHistogramVec(stream *util.Stream) {
 		// instance always set to the empty string
 		[]string{"app", "env", "action", "instance"},
 	)
-	c.registry.MustRegister(c.ajaxHistogramVec)
+	c.registry.MustRegister(c.actionMetrics.ajaxHistogramVec)
+	c.applicationMetrics.ajaxHistogramVec = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "logjam:application:ajax_time_distribution_seconds",
+			Help:    "logjam ajax response time distribution by application",
+			Buckets: stream.AjaxBuckets,
+		},
+		// instance always set to the empty string
+		[]string{"app", "env", "instance"},
+	)
+	c.registry.MustRegister(c.applicationMetrics.ajaxHistogramVec)
 }
 
-func (c *Collector) registerActionCallsTotalVec() {
-	c.actionCallsTotalVec = prometheus.NewCounterVec(
+func (c *Collector) registerActionsProcessedTotalVec() {
```

Comment:
We could rename the function to `registerTransactionsTotalVec` to keep it in line with the other ones.

Reply:
fixed